### ISSUE TITLE
How to disable the formatting for the helm chart name ?

### ISSUE THREAD

**Issue by raul1991** [rapidsai/frigate#15]
First of all, Damn! this tool is fantastic just does what it says, perfectly.

Now, the issue is that the helm chart initial character gets rendered as upper case. Is there a flag to disable it ? 

Example:

```
description: Helm chart for foo-bar
name: foo-bar
version: 0.9.0-15
```
Expectation
------------------
Chart name to be rendered as is

Actual
--------------
chart name is rendered as Foo-bar

**Comment by raul1991** [comment]
Ok, I see this [link](https://frigate.readthedocs.io/en/master/customizing.html#extending-built-in-templates) and maybe overriding a template variable {{name}} would help here but where should I place it or tell frigate about it ? .frigade ?

**Comment by raul1991** [comment]
I guess I was too fast in creating the issue because after reading the docs I got this working. So put a file like this in your helm chart to achieve the above

```
{% extends "markdown.jinja2" %}

{% block title -%}
# {{ name }}
{%- endblock %}

```

**Comment by jacobtomlinson** [comment]
Thanks for raising this @raul1991 and thanks for the praise!

I think this may have been fixed by #14.

**Comment by raul1991** [comment]
I think the pull request removed the support for capitalization of the comments.

This issue was regarding the capitalization of the chart name inthe rendered output file. This could be done by overriding the respective template inside a .frigate file. I didn't see it that time so raised an issue but lets close this now.
